src/Bicep.Core.Samples/Files/TypeDeclarations_LF/main.json

```diff
@@ -7,7 +7,7 @@
     "_generator": {
       "name": "bicep",
       "version": "dev",
-      "templateHash": "16153089583935393416"
```

Comment:
Talked through this one with @anthony-c-martin. The value that gets hashed [gets parsed as an ARM template object](https://github.com/Azure/bicep/blob/01448af05d304b5eada196d2f63e446bb289e971/src/Bicep.Core/Emit/TemplateWriter.cs#L136) and then [reserialized as JSON](https://github.com/Azure/bicep/blob/01448af05d304b5eada196d2f63e446bb289e971/src/Bicep.Core/Emit/TemplateWriter.cs#L93), which will silently drop unknown properties. Upgrading Azure.Deployments.Templates therefore caused additional properties in this file to be included in the hashed value, so the hash change is expected.